packages/fuel-indexer-lib/src/utils.rs

```diff
@@ -67,36 +66,40 @@ pub fn local_repository_root() -> Option<String> {
     Some(root_dir)
 }
 
+/// Request to reload the specified indexer executor using this indexer's current
+/// assets in the database.
+///
+/// Sent from API server to indexer service.
 #[derive(Debug)]
-pub struct AssetReloadRequest {
+pub struct ReloadRequest {
     pub namespace: String,
     pub identifier: String,
 }
 
+/// Request to remove the specified indexer executor from the indexer service.
+///
+/// Sent from API server to indexer service.
```

Comment:
We can probably remove this line as it's already mentioned in the doc comment for `ServiceRequest`.